extensions/big-data-cluster/src/bigDataCluster/controller/clusterControllerApi.ts

```diff
@@ -203,6 +247,28 @@ export interface IBdcStatusResponse {
 	bdcStatus: BdcStatusModel;
 }
 
+export enum MountState {
```

Comment:
I am also not sure how to choose between enum and union type( 'Creating' | 'Ready'|'Error'), do we have any guideline for that?

Reply:
I'm really not sure - this one 'felt' like an enum to me, but happy to align on a common standard if folks have one? I'm thinking union type can be helpful if you have to lowercase / convert it?